cpp/oneapi/dal/test/engine/gtest_main.cpp

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
```

Comment:
Do we plan to delete this when all tests will be moved to Catch2?

Reply:
Yes, but it will happen in the next PR